frontend/textbook.ts

```diff
@@ -65,9 +65,12 @@ textbook.runAfterDOMLoaded(() => {
   initIndexHighlight()
 
   // set up & trigger segment
-  initAnalytics(window.textbookAnalytics.key, window.textbookAnalytics.url)
-  textbook.trackClickEvent = trackClickEvent
-  textbook.trackPerformedSearch = trackPerformedSearch
-  textbook.trackUpdatedObject = trackUpdatedObject
-  trackPage(window.location.pathname)
+  initAnalytics(
```

Comment:
@vabarbosa - this is awesome, thank you! 

I was wondering if you've been able to confirm that this'll be tracking correctly, after this change?

Reply:
i am not sure if we can get tracking from localhost or from the preview link since the domains are not what is expected. we may need to get confirmation from Anne once deployed. i will do a little digging around and see if there is way to confirm.

Reply:
Okay, thanks for the response, I don't think we need to block this. Let's confirm once it's deployed to ensure we're capturing what we need.

🚢